ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```
@@ -75,6 +75,24 @@ fi
 export EXTRA_FLAGS UUID
 export ADDITIONAL_PARAMS
 
+# Setup pprof secrets for kube-apiserver profiling
+oc create ns benchmark-operator || true
+oc create serviceaccount kube-burner -n benchmark-operator || true
+oc create clusterrolebinding kube-burner-crb --clusterrole=cluster-admin --serviceaccount=benchmark-operator:kube-burner || true
+export BEARER_TOKEN=$(oc create token -n benchmark-operator kube-burner --duration=6h || oc sa get-token kube-burner -n benchmark-operator)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)